commcare_connect/opportunity/helpers.py

```diff
@@ -20,14 +20,19 @@ def get_annotated_opportunity_access(opportunity: Opportunity):
                 )
             ),
             completed_modules_count=Count(
-                "user__completed_modules", filter=Q(user__completed_modules__opportunity=opportunity)
+                "user__completed_modules",
+                filter=Q(user__completed_modules__opportunity=opportunity),
+                distinct=True,
             ),
+            job_claimed=Case(When(Q(opportunityclaim__isnull=False), then="opportunityclaim__date_claimed")),
```

Comment:
Why add this instead of what we had before?

Reply:
The total column tries to resolve a related object i.e opportunityclaim in this case which could also be None leading to no related object found error. So i added this aggregation which tries to avoid using django to get the foreign key to opportunityclaim and does it in aggregation instead.